modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -19,90 +20,251 @@ To configure a connector in Anypoint Studio:
+** Password +
+The password for the truststore file.
+** Type +
+The file format of the truststore file.
+** Algorithm +
+The algorithm the trust store uses. 
+** Insecure +
+Boolean that determines whether or not to validate the truststore. If set to true, no validation occurs. The default value is false.
+* *Key Store Configuration*
+** Type +
+Optionally specify the file format of the keystore file. The default value is `JKS`.
+** Path +
+The location of the keystore file. This is optional and can be used for two-way authentication for the connector.
+** Alias +
+When the keystore contains many private keys, this attribute indicates the alias of the key to use. If not defined, the first key in the file is used by default.
+** Key password +
+The key manager password, which is the password for the private key inside the keystore.
+** Password +
+The store password for the keystore file. This is optional and needed only if the *Key Store Location* is configured.
+** Algorithm +
+The algorithm used in the keystore. 
 +
-.. Kerberos:
-** All the parameters from the Basic configuration.
-** Principal - Kerberos principal
-** Keytab - Path to keytab file associated with the `principal`.
-** Service Name - The Kerberos principal name that the Kafka broker runs as.
-** Additional JAAS Properties - Additional properties as key-value pairs that you set in `sasl.jaas.config` and that you usually include in JAAS configuration file.
+image::kafka/kafka-tls-studio-config.png[]
+
+== Configure the Commit Operation
+
+. Drag the *Commit* operation to the Studio canvas.
+. Configure the *Commit* operation in the *General* tab:
 +
-image::kafka/kafka-kerberos-studio-config.png[]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Consumer commit key a| String |  The commitKey of the last poll. This operation is valid only when used inside a flow that is using one of the listener sources (*Batch message listener* or *Message listener*) which inserts this value as an attribute in the Mule Event. |  | x
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
 +
-.. Kerberos SSL:
-** All the parameters from the Basic configuration.
-** All the parameters from the SSL configuration.
-** All the parameters from the Kerberos configuration.
+image::kafka/kafka-commit-studio-config-general.png[]
+
+== Configure the Consume Operation
+
+. Drag the *Consume* operation to the Studio canvas.
+. Configure the *Consume* operation in the *General* tab:
 +
-image::kafka/kafka-kerberos-ssl-studio-config.png[]
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Consumption timeout a| Number |  The amount of TimeUnits that this operation will wait for receiving messages. |  | 
```

Comment:
TimeUnits is not a field, at least that I can see. I would change this to "time units."